Examples/Mocks/Info.md

```diff
@@ -0,0 +1,41 @@
+## AutoMockable
```

Comment:
I really like pattern for mocks used in [Dobby](https://github.com/trivago/Dobby), so wanted to write a template for it some time too